clang/test/CIR/CodeGen/AArch64/neon.c

```diff
@@ -858,7 +858,7 @@ int16x4_t test_vabd_s16(int16x4_t v1, int16x4_t v2) {
   return vabd_s16(v1, v2);
 
   // CIR-LABEL: vabd_s16
-  // CIR: {{%.*}} = cir.llvm.intrinsic "aarch64.neon.sabd" {{%.*}}, {{%.*}} : 
+  // CIR: {{%.*}} = cir.llvm.intrinsic "aarch64.neon.sabd" {{%.*}}, {{%.*}} :
```

Comment:
Nit: please fix your IDE to not touch lines you are not directly editing, this generates a lot of noise and polute git blame - if you feel brave about cleaning up those things should come into their own NFC PRs.